### ISSUE TITLE
Stopped working with the latest insider builds

### ISSUE THREAD

**Issue by tuhlmann** [rafamel/subtle-brackets#7]
Hi,
I was using that great extension for a few weeks now.
A few days ago it stopped working though. I don't see an error, just no matching indicator appears.
I'm using the latest vscode insider build that is updated daily, so I can't sadly point to the latest working release. The version I'm currently using is:

Version 1.22.0-insider
Commit ec3c083444c084394ee0cd936074019c1bc0b321
Datum 2018-03-14T06:14:24.748Z
Shell 1.7.9
Renderer 58.0.3029.110
Node 7.9.0
Architektur x64

OS is Ubuntu 17.10.

Is there any workaround to get it back working or to give you more info?

Thanks,
Torsten.

**Comment by tuhlmann** [comment]
Here's the error I get in the developer console:

```
Error activating extension "rafamel.subtle-brackets": TypeError: Cannot assign to read only property 'borderColor' of object.
```

**Comment by rafamel** [comment]
Hi there, Torsten, I finally had some time to look this error over. I haven't been able to test it out on v1.22, but I have a theory as to what is causing it. If it's what I think, the PR #9 wouldn't solve the underlying problem. Could you please test out on your computer the branch fix-readonly of this repo and let me know if the error still occurs? 

Thanks for the help!
Rafa.

**Comment by rafamel** [comment]
This is now fixed on v2.1.1 by taking a different approach.
